nvme.c

```diff
@@ -164,7 +164,7 @@ void *nvme_alloc(size_t len, bool *huge)
 
 static ssize_t getrandom_bytes(void *buf, size_t buflen)
 {
-#if !HAVE_SYS_RANDOM
```

Comment:
Ooh, my bad. This was from my testing.

Reply:
since we find it fast, doesn't matter :)